src/SDK/Logs/ReadableLogRecord.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Logs;
+
+use OpenTelemetry\API\Logs\LogRecordData;
+use OpenTelemetry\Context\ContextInterface;
+use OpenTelemetry\SDK\Common\Attribute\AttributesInterface;
+use OpenTelemetry\SDK\Common\Instrumentation\InstrumentationScopeInterface;
+use OpenTelemetry\SDK\Resource\ResourceInfo;
+
+class ReadableLogRecord
```

Comment:
I'm wondering, from engineering point of view, why this class is not derived from `Api::LogRecord`? We could prohibit write operations by throwing exceptions and we could get rid of `LogRecordData` type which is not mentioned in the spec and seems artificial. I see that different implementation do that differently and java follows the same pattern.

Reply:
Agreed, I've reworked this part a bit, and removed `LogRecordDate` - I think it's closer to spec now.